clientd3d/d3drender.c

```diff
@@ -8237,13 +8262,13 @@ void D3DRenderPlayerOverlaysDraw(d3d_render_pool_new *pPool, room_type *room, Dr
 			pChunk->st0[3].t = oneOverH;
 		}
 
-		pChunk->st1[0].t -= (gFrame & 3) / 256.0f;

```

Comment:
This is very odd.. st1[0].s never gets set.  I think this is a longstanding bug?  Or is it somehow intentional?